pkg/snapshotter/snapshotter.go

```diff
@@ -138,9 +135,6 @@ func (s *snapshot) GetSnapshotStatus(ctx context.Context, snapshotID string, sna
 		return false, time.Time{}, 0, fmt.Errorf("can not find snapshot for snapshotID %s", snapshotID)
 	}
 
-	creationTime, err := ptypes.Timestamp(rsp.Entries[0].Snapshot.CreationTime)
-	if err != nil {
-		return false, time.Time{}, 0, err
-	}
+	creationTime := rsp.Entries[0].Snapshot.CreationTime.AsTime()
```

Comment:
Same as above.

Reply:
Response same as above, AsTime() doesn't return a nillable type or an error to check.